web/src/components/GuidanceListPage/GuidanceListPage.tsx

```diff
@@ -138,6 +143,7 @@ export const getGuidanceListPage =
 				<PageHeader
 					preheading={preheading}
 					heading={heading}
+					id="content-start"
```

Comment:
I think we also need to add the id to https://github.com/nice-digital/next-web/blob/main/web/src/components/ErrorPageContent/ErrorPageContent.tsx#L38 now that it's not on `main`